src/IssueProcessor.ts

```diff
@@ -139,10 +154,22 @@ export class IssueProcessor {
       const skipMessage = isPr
         ? this.options.skipStalePrMessage
         : this.options.skipStaleIssueMessage;
-      const issueType: string = isPr ? 'pr' : 'issue';
-      const shouldMarkWhenStale = this.options.daysBeforeStale > -1;
+      const issueType: IssueTypeEnum = getIssueType(isPr);
+      const DAYS_BEFORE_STALE: number = isPr
+        ? this._getDaysBeforePrStale()
+        : this._getDaysBeforeIssueStale();
+
+      if (isPr) {
+        core.info(`Days before pull request stale: ${DAYS_BEFORE_STALE}`);
+      } else {
+        core.info(`Days before issue stale: ${DAYS_BEFORE_STALE}`);
+      }
+
+      const SHOULD_MARK_WHEN_STALE: boolean = shouldMarkWhenStale(
```

Comment:
nit: can we change these const declarations to not be all caps and follow the same standard as the other consts? (camel case).